jck/build.xml

```diff
@@ -92,31 +92,42 @@
 			<available file="${JCK_ROOT_USED}" type="dir" />
 			<then>
 				<!--Obtain local and remote SHA to figure out if local materials are up-to-date--> 
-				<exec executable="git" dir="${JCK_ROOT_USED}" outputproperty="localSHA">
+				<exec executable="git" dir="${JCK_ROOT_USED}" failonerror="false" outputproperty="localSHA" resultproperty="return.code">
 					<arg line="rev-parse"/>
 					<arg line="HEAD"/>
 				</exec>
-				<exec executable="git" dir="${JCK_ROOT_USED}" outputproperty="remoteSHA">
-					<arg line="ls-remote"/>
-					<arg line="${JCK_GIT_REPO_USED}"/>
-					<arg line="${jck_branch}"/>
-				</exec>
-				<loadresource property="localSHATrimmed">
-					<propertyresource name="localSHA"/>
-					<filterchain>
-						<tokenfilter>
-							<trim/>
-						</tokenfilter>
-						<striplinebreaks/>
-					</filterchain>
-				</loadresource>
-					
-				<echo message="LocalSHA = --${localSHATrimmed}--"/>
-				<echo message="RemoteSHA= --${remoteSHA}--"/>
+				<if>
+					<not>
+						<equals arg1="${return.code}" arg2="0"/>
+					</not>
+					<then>
+						<echo message="fatal: not a git repository.... Delete the directory and retry" />
+						<delete dir="${JCK_ROOT_USED}" failonerror="false" />
+					</then>
```

Comment:
I suggest a more generic error message in this case since the error check is for anything non-zero.  I believe we also want to fail on error for deleting the directory, otherwise if there is a failure with the existing repo it will just resume and fail in the update materials section with the same error.

Reply:
Changed to more generic error message and changed fail on error to true for deleting the directory.

Reply:
Technically its not "retry" but "continue".  Other than that I believe this does make recovery in a "corrupt" local cache possible and better than before.  While I can approve I should note that I am not a regular Adopium reviewer/committer that is intimately familiar with how this might have a larger impact.  @llxia can you please also advise.

Reply:
Changed to `continue`. Thanks